frontend/src/scenes/authentication/inviteSignupLogic.ts

```diff
@@ -76,14 +77,15 @@ export const inviteSignupLogic = kea<inviteSignupLogicType>([
     })),
     forms(({ actions, values }) => ({
         signup: {
-            defaults: { email_opt_in: true } as AcceptInvitePayloadInterface,
-            errors: ({ password, first_name }) => ({
+            defaults: { email_opt_in: true, role_at_organization: '' } as AcceptInvitePayloadInterface,
+            errors: ({ password, first_name, role_at_organization }) => ({
                 password: !password
                     ? 'Please enter your password to continue'
                     : password.length < 8
                     ? 'Password must be at least 8 characters'
                     : undefined,
                 first_name: !first_name ? 'Please enter your name' : undefined,
+                role_at_organization: !role_at_organization ? 'Please enter your role' : undefined,
```

Comment:
Curious why this is a non-optional field?

Reply:
We're interested in using it for the onboarding - if someone says they are in Engineering, we can just jump them straight to implementation details. If they say they are something else, we might recommend other steps. But it could definitely be optional..